packages/checked_oint/checked_oint.0.1.0/opam

```diff
@@ -0,0 +1,41 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "An OCaml library for checked integer arithmetic"
+maintainer: ["Tima Kinsart <hirrolot@gmail.com>"]
+authors: ["Tima Kinsart <hirrolot@gmail.com>"]
+license: "MIT"
+homepage: "https://github.com/Hirrolot/checked_oint"
+bug-reports: "https://github.com/Hirrolot/checked_oint/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.14"}
+  "ctypes-foreign"
```

Comment:
```suggestion
  "ctypes"
  "ctypes-foreign"
```

Reply:
Let's try this

Reply:
The issue still persists on `5.2~beta2`.